Add blog post: Enabling upgrades to x86_64_v2 with ELevate NG (#988)
AlmaLinux/almalinux.org: content/blog/2026-04-15-elevate-ng-to-x86_64_v2.md

---

> title: "Enabling upgrades to AlmaLinux 10 x86_64_v2 with ELevate NG"

Reviewer: ```suggestion
title: "Enabling ELevateing to AlmaLinux 10 x86_64_v2"
```
I want to make ELevate a more integral part of this title. Does this sound okay to you, @ykohut ? It's not at all proper english, but I think it expressses what we need it to.

Author: ```suggestion
title: "ELevating to AlmaLinux 10 x86_64_v2"
```

Author: What about as simple expression ... ?

Reviewer: YES! that's perfect.

type: blog
author:
  name: "Yuriy Kohut"
  bio: "ELevate Project Engineer"
  image: /users/yuriy-kohut.jpg
date: 2026-04-15
images:
  - /blog-images/2026/2026-04-15-elevate-ng-to-x86_64_v2.png
post:
  title: "ELevate NG now supports upgrading from AlmaLinux 9 to AlmaLinux 10 (or AlmaLinux Kitten) x86_64_v2. Help us verify the upgrade!"
  image: /blog-images/2026/2026-04-15-elevate-ng-to-x86_64_v2.png
---

> ELevate NG now enables the upgrade path from AlmaLinux 9 to AlmaLinux 10 (or AlmaLinux Kitten) x86_64_v2. We need your help to verify that the upgrade works smoothly — please give it a try and report your results!

Reviewer: ```suggestion
ELevate NG (the development branch of ELevate) now supports the upgrade path from AlmaLinux OS 9 to AlmaLinux OS 10 (or AlmaLinux OS Kitten) x86_64_v2. We need your help to verify that the upgrade works smoothly. Please give it a try and report your results!
```
